code_samples/back_office/udw/assets/js/image-tab/image.tab.module.js

```diff
@@ -1,28 +1,28 @@
 import React, { useContext } from 'react';
-    
-    import Tab from '../../../vendor/ezsystems/ezplatform-admin-ui/src/bundle/ui-dev/src/modules/universal-discovery/components/tab/tab';
-    import ImagesList from './components/images.list';
-    
-    const ImageTabModule = () => {
-        return (
-            <div className="m-image-tab">
-                <Tab isContentOnTheFlyDisabled={true} isSortSwitcherDisabled={true} isViewSwitcherDisabled={true}>
-                    <ImagesList />
-                </Tab>
-            </div>
-        );
-    };
+
+import Tab from '@ibexa-admin-ui/src/bundle/ui-dev/src/modules/universal-discovery/components/tab/tab';
+import ImagesList from './components/image.list';
```

Comment:
```suggestion
import ImagesList from './components/images.list';
```
This actually worked forme only as `images`